docs/copilot/copilot-customization.md

```diff
@@ -161,77 +251,120 @@ Common use cases include:
     * Customize UX-friendly validation rules
     ```
 
-* Document reusable security practices for REST APIs, which can be used to do security reviews of REST APIs:
+* Prompt for performing a security review of a REST API:
 
     ```markdown
-    Secure REST API review:
+    ---
+    mode: 'edit'
+    description: 'Perform a REST API security review'
+    ---
+    Perform a REST API security review:
+
     * Ensure all endpoints are protected by authentication and authorization
     * Validate all user inputs and sanitize data
     * Implement rate limiting and throttling
     * Implement logging and monitoring for security events
-    …
     ```
 
+> [!TIP]
+> Reference additional context files like API specs or documentation by using Markdown links to provide Copilot with more complete information.
+
 ### Create a workspace prompt file
 
-Workspace prompt files are stored in your workspace and are only available in that workspace. Workspace prompt files are stored in the `.github/prompts` directory of your workspace.
+Workspace prompt files are stored in your workspace and are only available in that workspace.
+
+By default, prompt files are located in the `.github/prompts` directory of your workspace. You can specify additional prompt file locations with the `setting(chat.promptFilesLocations)` setting.
 
 To create a workspace prompt file:
 
-1. Set the `setting(chat.promptFiles)` setting to `true` for the `.github/prompts` directory.
+1. Run the **Chat: New Prompt File** command from the Command Palette (`kb(workbench.action.showCommands)`).
+
+1. Choose the location where the prompt file should be created.
 
-1. Create a `.prompt.md` file in the `.github/prompts` directory of your workspace.
+    By default, only the `.github/prompts` folder is available. Add more prompt folders for your workspace with the `setting(chat.promptFilesLocations)` setting.
 
-    By default, prompt files are located in the `.github/prompts` directory of your workspace. You can specify additional prompt file locations with the `setting(chat.promptFilesLocations)` setting.
+1. Enter a name for your prompt file.
```

Comment:
Would it make sense to provide an opinionated naming example?

Reply:
Not sure about this, as this is mostly user-specific or scoped to the workspace. Not convinced we can come up a convention that meets the very broad range of scenarios that can be covered.